src/resources/js/move-tickets-dialog.js

```diff
@@ -248,17 +248,32 @@ var tribe_move_tickets = tribe_move_tickets || {};
 
 				// Clear the existing list
 				$post_choices.html( '' );
-				let total_posts = 0;
-
-				for ( const key in response.data.posts ) {
-					const post_id = parseInt( key, 10 );
-					const title = response.data.posts[ key ];
-					total_posts++;
-
-					$post_choices.append(
-						'<label> <input type="radio" value="' + post_id + '" name="post-choice">' + title + '</label>'
-					);
-				}
+				var total_posts = 0;
+                
+                // Convert the object to a sortable array
```

Comment:
This needs to be using tabs instead of spaces for indentation.

Also, all of the new JS code should follow our coding standards (spaces within parentheses, alignment, etc.).